uber/sep_commands.py

```diff
@@ -2,6 +2,88 @@
 
 
 @entry_point
+def alembic(*args):
+    """
+    Frontend for alembic script with additional uber specific facilities.
+
+    "sep alembic" supports all the same arguments as the regular "alembic"
+    command, with the addition of the "--plugin PLUGIN_NAME" option.
+
+    Passing "--plugin PLUGIN_NAME" will choose the correct alembic version path
+    for the given plugin. If "--plugin" is omitted, it will default to "uber".
+
+    If "--version-path PATH" is also specified, it will override the version
+    path chosen for the plugin. This functionality is rarely needed, and best
+    left unused.
+
+    If a new migration revision is created for a plugin that previously did
+    not have any revisions, then a new branch label is applied using the
+    plugin name. For example::
+
+        sep alembic --plugin new_plugin revision --autogenerate -m "Initial migration"
+
+    A new revision script will be created in "new_plugin/alembic/versions/"
+    with a branch label of "new_plugin". The "new_plugin/alembic/versions/"
+    directory will be created if it does not already exist.
+
+    If "--branch-label LABEL" is also specified, it will override the plugin
+    name as the new branch label.
+    """
+    argv = args if args else sys.argv[1:]
+
+    # Extract the "--plugin" option from argv.
+    plugin_name = 'uber'
+    for plugin_opt in ('-p', '--plugin'):
+        if plugin_opt in argv:
+            plugin_index = argv.index(plugin_opt)
+            argv.pop(plugin_index)
+            plugin_name = argv.pop(plugin_index)
+
+    from glob import glob
+    from os.path import exists, join
+    from alembic.config import CommandLine
+    from sideboard.config import config as sideboard_config
+    from uber.migration import create_alembic_config, version_locations
+
+    assert plugin_name in version_locations, (
+        'Plugin "{}" does not exist in {}'.format(
+            plugin_name, sideboard_config['plugins_dir']))
+
+    commandline = CommandLine(prog='sep alembic')
+    if any([h in argv for h in ('-h', '--help')]):
+        # If "--help" is passed, add a description of the "--plugin" option
+        commandline.parser.add_argument(
+            '-p', '--plugin',
+            type=str,
+            default='uber',
+            help='Plugin in which to add new versions')
```

Comment:
Help text could be expanded here, specifically wrt what format the plugin name should be. If I'm reading it right it should be based on the folder name of the plugin inside `/sideboard/plugins/` but it would be nice to say, like, `For example: To generate a migration for a plugin /sideboard/plugins/plugin_name, use '--plugin plugin_name'`.

Additionally, it looks like this only talks about the `--plugin` flag but there are others. This is the text that gets displayed when you type `sep alembic --help`, right? The output would ideally look a lot more like the (very thorough) docstring.